src/custom/pages/Claim/ClaimingStatus.tsx

```diff
@@ -0,0 +1,70 @@
+import { Trans } from '@lingui/macro'
```

Comment:
Moves html code from index into this file, adds global claim state null checks

commented out is claimedAmount which is formatted incorrectly. fix incoming